python/plugins/processing/algs/gdal/OgrToPostGis.py

```diff
@@ -234,18 +241,18 @@ def getConsoleCommands(self, parameters, context, feedback, executing=True):
             arguments.append('"' + shapeEncoding + '"')
         arguments.append('-f')
         arguments.append('PostgreSQL')
-        arguments.append('PG:"')
-        arguments.append(self.getConnectionString(parameters, context))
-        arguments.append('"')
+        arguments.append('PG:'+self.getConnectionString(parameters, context))
         arguments.append(dimstring)
-        arguments.append(ogrLayer)
-        arguments.append(layername)
+        arguments.append('"{0}"'.format(ogrLayer))
```

Comment:
Is this change required? It should already be quoted if needed, and there's a chance this will cause double-quotes for some layer names